src/parser/jobs/brd/PitchPerfectUsage.module.css

```diff
@@ -0,0 +1,19 @@
+.title {
```

Comment:
This file isn't being imported anywhere - did you forget to update the `Barrage.module.css` import in `PitchPerfectUsage`?